javascript/packages/cli/src/cli.ts

```diff
@@ -87,21 +87,24 @@ const downloadBinaries = async (binaries: string[]): Promise<void> => {
           const progressBar = multibar.create(parseInt(contentLength, 10), 0);
           const reader = response.body?.getReader();
           const writer = fs.createWriteStream(path.resolve(name));
-
-          while (true) {
-            const read = await reader?.read()!;
-            if (read?.done) {
-              writer.close();
-              resolve();
-              break;
+          let i = true;
+          while (i) {
+            const read = await reader?.read();
+            if (read) {
+              if (read?.done) {
+                writer.close();
+                i = false;
+                resolve();
+                break;
```

Comment:
If we use `i` as condition we don't need to `break` here. But also this is more `explicit` 🤔

Reply:
using a const for a while loop is fundamentally wrong, and could lead to eternal loop.. I understand your mentions but I prefer to remove the the break